ui/src/logs/components/SeverityColumnFormat.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {SFC} from 'react'
+
+import {SeverityFormatOptions} from 'src/logs/constants'
+import {SeverityFormat} from 'src/types/logs'
+
+interface Props {
+  format: SeverityFormat
+  onChangeFormat: (format: SeverityFormat) => () => void
```

Comment:
Is there any way we can accomplish this without a function closure? We have had some leaky state issues caused by these in other parts of the app.